x-pack/plugins/cases/docs/openapi/components/schemas/create_case_request.yaml

```diff
@@ -0,0 +1,11 @@
+title: Create case request
+description: >-
+  The create case API request body varies depending on the type of connector.
+oneOf:
```

Comment:
Would it be better to have a single `create_case_connector.yaml` file and then embed the `oneOf` in there with all these refs?

Reply:
I'll play around with that today to see whether that would pass the validation checks

Reply:
I've made this change in 636f24254f3bcd83cfd21db1328ecf260a78120b